### PR TITLE
feat: added BTBS ERC20 tokens

### PR DESCRIPTION
**Pull request by GuillermoBlasco**
Added config for fixprice, eth extension and cryptx wallet

### CODE REVIEW

File: server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/wallets/cryptx/v2/CryptXWallet.java [hunk 1]
```diff
@@ -110,7 +110,7 @@ public String getCryptoAddress(String cryptoCurrency) {
         coins.add(CryptoCurrency.BCH.getCode());
         coins.add(CryptoCurrency.ETH.getCode());
         coins.add(CryptoCurrency.USDT.getCode());
-
+        coins.add(CryptoCurrency.BTBS.getCode());
```
Comment (pvyhnal-generalbytes) [inline server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/wallets/cryptx/v2/CryptXWallet.java:113]: Does CryptX support BTBS? Their documentation does not mention it: 
https://api.cryptx.com/#section/Coin-Digital-Currency-Support

File: server_extensions_extra/src/main/resources/batm-extensions.xml [hunk 2]
```diff
@@ -72,6 +72,7 @@
                 <cryptocurrency buyonly="true">BCH</cryptocurrency>
                 <cryptocurrency>BTC</cryptocurrency>
                 <cryptocurrency>LTC</cryptocurrency>
+                <cryptocurrency buyonly="true">BTBS</cryptocurrency>
```
Comment (pvyhnal-generalbytes) [inline server_extensions_extra/src/main/resources/batm-extensions.xml:75]: Are you adding two-way (buy & sell) or just one way (buy only) support for BTBS?
Here you specify "buyonly=true" and in that case you do not need to implement the "PaymentSupport" interface.

File: server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/ethereum/erc20/btbs/BtbsPaymentSupport.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+ *
+ ************************************************************************************/
+package com.generalbytes.batm.server.extensions.extra.ethereum.erc20.btbs;
+
+import com.generalbytes.batm.common.currencies.CryptoCurrency;
+import com.generalbytes.batm.server.extensions.extra.common.QueryableWalletPaymentSupport;
+
+import java.util.concurrent.TimeUnit;
+
+public class BtbsPaymentSupport extends QueryableWalletPaymentSupport {
```
Comment (pvyhnal-generalbytes) [inline server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/ethereum/erc20/btbs/BtbsPaymentSupport.java:25]: QueryableWalletPaymentSupport would only work with coinbase wallet for now

**Review by pvyhnal-generalbytes — CHANGES_REQUESTED** [review]
I added a few comments regarding two-way (buy & sell) support. 
If you only need Buy support, Infura wallet should be enough and you don't need the PaymentSupport implementation.

The logos provided do not follow the style of the other crypto currency logos on the ATM, please see template.svg

**Comment by GuillermoBlasco** [comment]
Thanks! We'll review it and update the PR.

**Comment by GuillermoBlasco** [comment]
Logos updated, and code fixed. Still waiting for Cryptx confirmation. Thanks pvyhnal-generalbytes

**Comment by GuillermoBlasco** [comment]
Hey @pvyhnal-generalbytes ! We have confirmation from Cryptx that BTBS is already supported! It is not listed in their API documentation because they support +100 ERC20 tokens, so they don't list them all. But we have confirmation.

Therefore, is it ready for review and merge? 

Thanks!

**Pull request merged**